pkg/k8s/ipresolver.go

```diff
@@ -20,6 +22,36 @@ import (
 
 const MAX_RESOLVED_DNS = 10000 // arbitrary limit
 
+var (
+	podsCounter = promauto.NewCounter(prometheus.CounterOpts{
```

Comment:
Use NewCounterVec instead - 1 series ("caretta_watcher_events_counter") with labels that describe the event type (e.g pods, nodes...)

Reply:
done